.swiftlint.yml

```diff
@@ -85,6 +85,8 @@ opt_in_rules: # some rules are only opt-in
   - return_arrow_whitespace
   - unused_import
   - vertical_parameter_alignment_on_call
+  - vertical_parameter_alignment
```

Comment:
This rule is in the `disabled_rules`, we should enable it by removing the disable instead.

Reply:
oh dang. Good catch. I had this in a stash from before and I forgot to apply it. Applied.